templates/generate.html

```diff
@@ -0,0 +1,12 @@
+<html>
+    <head>
+        <title>Hack Coins | Admin Generate QR Code</title>
+    </head>
+    <body>
+        <h2>Welcome, @{{ name }}!</h2>

```

Comment:
is {{ name }} necessarily going to be fulfilled by their slack ID? 

If it is just resolve this